src/Bridge/Symfony/Validator/EventListener/ValidateListener.php

```diff
@@ -82,6 +83,10 @@ public function onKernelView(ViewEvent $event): void
             $validationGroups = $validationGroups($data);
         }
 
+        if ($this->container instanceof SymfonyContainerInterface && empty($validationGroups)) {
```

Comment:
You are adding a feature to deprecated class :guardsman:

Reply:
Oh you're right ! I didn't see that. I will fix that.